### PR TITLE
refactor(@ui/settings): add device syncing section to storybook; fix enable device bug

### PR DESCRIPTION
**Pull request by iurimatias**
### What does the PR do

Add the device settings section to storybug. also fixes a bug with enable device, found while using storybook.

### CODE REVIEW

**Review by borismelnik — APPROVED** [review]
LGTM :)

File: storybook/pages/DevicesSectionPage.qml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                        }
+
+                        Flow {
+                            Layout.fillWidth: true
+
+                            CheckBox {
+                                text: "enabled"
+                                checked: model.enabled
+                                onToggled: model.enabled = !model.isCurrentDevice
+                            }
```
Comment (caybro) [inline storybook/pages/DevicesSectionPage.qml:168]: LGTM but what is it with the Flow around every CheckBox? :)

**Review by caybro — APPROVED** [review]

**Review by noeliaSD — CHANGES_REQUESTED** [review]
Something looking strange here ;) : 

Screenshot 2022-10-18 at 15 49 48

File: ui/app/AppLayouts/Profile/views/DevicesView.qml [hunk 2]
```diff
@@ -17,6 +17,7 @@ import "../stores"
 SettingsContentBase {
     id: root
 
+    property var devicesList
```
Comment (noeliaSD) [inline ui/app/AppLayouts/Profile/views/DevicesView.qml:20]: Not sure about the need of this property. Why not use directly the `store.deviceModel`?

**Pull request merged**